### PR TITLE
Kamu UI 398 fixes env vars and secrets for dataset flows

### PR DESCRIPTION
**Pull request by dmitriy-borzenko**
Closes: https://github.com/kamu-data/kamu-web-ui/issues/398

### CODE REVIEW

File: src/app/api/kamu.graphql.interface.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // THIS FILE IS GENERATED, DO NOT EDIT!
-import { gql } from "apollo-angular";
+import { gql } from "@apollo/client/core";
```
Comment (zaychenko-sergei) [inline src/app/api/kamu.graphql.interface.ts:2]: Oops

File: src/app/dataset-view/additional-components/dataset-settings-component/tabs/compacting/dataset-settings-compacting-tab.component.ts [hunk 2]
```diff
@@ -37,9 +44,31 @@ export class DatasetSettingsCompactingTabComponent extends BaseComponent {
         private fb: FormBuilder,
         private datasetCompactionService: DatasetCompactionService,
         private navigationService: NavigationService,
+        private datasetSchedulingService: DatasetSchedulingService,
     ) {
         super();
     }
+    ngOnInit(): void {
```
Comment (zaychenko-sergei) [inline src/app/dataset-view/additional-components/dataset-settings-component/tabs/compacting/dataset-settings-compacting-tab.component.ts:51]: Missing blank line before

File: src/app/dataset-view/additional-components/dataset-settings-component/tabs/compacting/dataset-settings-compacting-tab.types.ts [hunk 3]
```diff
@@ -9,3 +9,13 @@ export const sliceSizeMapper: Record<SliceUnit, number> = {
...
+
+export function sliceSizeMapperReverse(sizeInBytes: number): { size: number; unit: SliceUnit } {
+    if (sizeInBytes % Math.pow(2, 30) === 0) {
+        return { size: sizeInBytes / Math.pow(2, 30), unit: SliceUnit.GB };
+    } else if (sizeInBytes % Math.pow(2, 20) === 0) {
+        return { size: sizeInBytes / Math.pow(2, 20), unit: SliceUnit.MB };
+    } else {
+        return { size: sizeInBytes / Math.pow(2, 10), unit: SliceUnit.KB };
+    }
+}
```
Comment (zaychenko-sergei) [inline src/app/dataset-view/additional-components/dataset-settings-component/tabs/compacting/dataset-settings-compacting-tab.types.ts:21]: I think this isn't specific to the component, and maybe moved somewhere to common utility code

File: src/app/dataset-view/additional-components/overview-component/overview.component.ts [hunk 4]
```diff
@@ -234,6 +234,15 @@ export class OverviewComponent extends BaseComponent implements OnInit {
         return !_.isNil(this.currentState?.overview.metadata.currentWatermark);
     }
 
+    public get showAddDataButton(): boolean {
+        return (
+            Boolean(this.currentState?.data.length) &&
+            this.isUserLogged &&
+            !this.currentState?.overview.metadata.currentPollingSource &&
+            this.datasetBasics.kind === DatasetKind.Root
```
Comment (zaychenko-sergei) [inline src/app/dataset-view/additional-components/overview-component/overview.component.ts:242]: Why I can't update the derived dataset using this button?

File: src/app/dataset-view/additional-components/dataset-settings-component/tabs/compacting/dataset-settings-compacting-tab.component.ts (hunk 2 above)
Comment (dmitriy-borzenko) [reply src/app/dataset-view/additional-components/dataset-settings-component/tabs/compacting/dataset-settings-compacting-tab.component.ts:51]: Added

File: src/app/dataset-view/additional-components/dataset-settings-component/tabs/compacting/dataset-settings-compacting-tab.types.ts (hunk 3 above)
Comment (dmitriy-borzenko) [reply src/app/dataset-view/additional-components/dataset-settings-component/tabs/compacting/dataset-settings-compacting-tab.types.ts:21]: Moved to app helpers

**Review by zaychenko-sergei — APPROVED** [review]

**Pull request merged**